drivers/mongodb-v3-driver/src/main/java/com/github/cloudyrock/mongock/driver/mongodb/v3/changelogs/runonce/MongockV3LegacyMigrationChangeLog.java

```diff
@@ -0,0 +1,26 @@
+package com.github.cloudyrock.mongock.driver.mongodb.v3.changelogs.runonce;
+
+import com.github.cloudyrock.mongock.ChangeLog;
+import com.github.cloudyrock.mongock.ChangeSet;
+import com.github.cloudyrock.mongock.driver.mongodb.v3.changelogs.LegacyService;
+import com.github.cloudyrock.mongock.migration.MongockLegacyMigration;
+import com.mongodb.client.MongoDatabase;
+import io.changock.driver.api.entry.ChangeEntry;
+import io.changock.driver.api.entry.ChangeEntryService;
+import io.changock.migration.api.annotations.NonLockGuarded;
+import io.changock.migration.api.annotations.NonLockGuardedType;
+
+import javax.inject.Named;
+
+@ChangeLog(order = "00001")
+public class MongockV3LegacyMigrationChangeLog extends LegacyService {
```

Comment:
Out of curiosity, why extend `LegacyService` here?  It's already constructing a separate `LegacyService` instance and delegating to that.

Reply:
good spotted. I was initially to create a supper class, but then I changed my mind and created a separated class. I forgot to remove it.

It will be removed or raise PR if you prefer ;)